tools/cachestat_example

```diff
@@ -0,0 +1,75 @@
+# ./cachestat -h
```

Comment:
this file is a duplicate of .txt. please remove it. Other than that, it looks great! Awesome tool. Thanks